modules/mod-pcm/GetAcidizerTags.h

```diff
@@ -0,0 +1,33 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  GetAcidizerTags.h
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+#pragma once
+
+#include "AcidizerTags.h"
+#include "sndfile.h"
```

Comment:
You might replace the first two with forward declarations of incomplete types.

Reply:
Works for the AcidizerTags, but with SNDFILE I get the `redifinition with different basic type` error, don't know why.